ml-agents/mlagents/trainers/agent_processor.py

```diff
@@ -137,7 +137,7 @@ def _process_step(
                 action_pre = None
             action_probs = stored_take_action_outputs["log_probs"][idx]
             action_mask = stored_decision_step.action_mask
-            prev_action = self.policy.retrieve_previous_action([global_id])[0, :]
+            prev_action = self.policy.retrieve_previous_action_single(global_id)
```

Comment:
Would you be opposed to renaming the method `retrieve_previous_action` and the old method to `retrieve_previous_actions`? Seems more descriptive and more similar to the methods in the LL-API.